src/skins/vector/css/_common.scss

```diff
@@ -332,3 +332,30 @@ textarea {
     border: 1px solid gray;
     background-color: darkred;
 }
+
+.mx_TintableSvgButton {
```

Comment:
I'm very surprised that the TintableSvgButton CSS was wrong, and i'm failing to see why this is needed?  What's the story? :)

Reply:
oh, having read the react-sdk stuff this makes a bit more sense. *thinks*

Reply:
okay, i'm still a bit confused - is this just because you can't put alt text on an `<object/>`? surely `cursor: pointer` could work on an `<object/>`?

Reply:
Yup, it's mainly because you can't put alt text on the svg. The path element of the svg also overrides cursor styling of the object / svg. 

We could do the styling overrides in custom CSS. However, it seems like a fairly standard occurrence that we would want tintable SVGs to behave in the same way as standard "img" elements, so why not just add it here, along with the title text overlay?

Reply:
ok, lgtm. still dubious about having two spans introduced though (as per other PR)

Reply:
> surely cursor: pointer could work on an <object/>?

(Just to clarify) Nope, that doesn't work, you'd have to apply the styling on the underlying path element(s)